{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -16,7 +16,14 @@ universal = 1
 
 [flake8]
 exclude = docs
-max-line-length = 120
+max-line-length = 88
```

Comment:
Does running `black` do this for us? I don't use it so actually don't know.

Reply:
Pretty much. You can keep it higher here if you want, but black parsing will fail so there isn't much point. It is well described here: https://github.com/psf/black#line-length

Reply:
Thanks for the explainer :)